source/adapters/level_zero/adapter.cpp

```diff
@@ -223,15 +236,29 @@ ur_adapter_handle_t_::ur_adapter_handle_t_()
 #else
     HMODULE processHandle = nullptr;
 #endif
-    GlobalAdapter->getDeviceByUUIdFunctionPtr =
-        (zes_pfnDriverGetDeviceByUuidExp_t)ur_loader::LibLoader::getFunctionPtr(
-            processHandle, "zesDriverGetDeviceByUuidExp");
-    GlobalAdapter->getSysManDriversFunctionPtr =
-        (zes_pfnDriverGet_t)ur_loader::LibLoader::getFunctionPtr(
-            processHandle, "zesDriverGet");
-    GlobalAdapter->sysManInitFunctionPtr =
-        (zes_pfnInit_t)ur_loader::LibLoader::getFunctionPtr(processHandle,
-                                                            "zesInit");
+
+    // Check if the user has enabled the default L0 SysMan initialization.
+    const int UrSysManInitEnabled = [] {
```

Comment:
Simple question, is there a way to reduce code duplication here? This is the same code block as above

Reply:
Oh, I see the subtle difference... Maybe if there was a better name here, to make it more obvious.. 
UrSysManApiInitEnable?

Reply:
sure, I can change that.

Reply:
Updated in latest to have a clear distinction between the variables.